docs/manual/src/docs/asciidoc/_includes/servlet/oauth2/oauth2-resourceserver.adoc

```diff
@@ -1258,6 +1258,29 @@ http
 Resolving the tenant by claim is similar to doing so by request material.
 The only real difference is the `toTenant` method implementation:
 
+[source,java]
```

Comment:
I think this snippet might be in the wrong place. Note the previous sentence, which is introducing the code snippet that is now below this one. I think we'd need to change the description a bit, too, for the reader to be able to follow.